Week1/hm/queries4.js

```diff
@@ -0,0 +1,39 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password : 'hyfpassword',
+    database: 'world'
+});
+
+connection.connect();
+
+const queries =[
+    // countries with population greater than 8 million
+    'SELECT name FROM country WHERE population>=8000000',
+
+    // countries that have “land” in their names
+    'SELECT name FROM country WHERE name LIKE "%lands"',
```

Comment:
All queries are OK except this one but I think it's just a typo and you meant:

```
SELECT name FROM country WHERE name LIKE "%land%"
```

:)